contrib/k8s/examples/mysql/mysqldb-dbms-only-instance..yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
```

Comment:
There's an extra . in this filename.

Also... any reason we can't put the dbms manifest and db-only manifest in the same file? May be more convenient for the user and it's a good way to show off that our broker is smart enough to sort the dependency out.

Reply:
Yep, we can do that. Applies across the three modules.